From: "Dr. David Alan Gilbert" <linux@treblig.org>
Date: Tue, 28 May 2024 22:56:40 +0100
Subject: scsi: qla2xxx: Remove unused struct 'scsi_dif_tuple'
Patch-mainline: v6.11-rc1
Git-commit: 96281dfa266d333522c004205acc5ff1e9e3a337
References: bsc#1228850

'scsi_dif_tuple' is unused since commit 8cb2049c7448 ("[SCSI] qla2xxx: T10
DIF - Handle uninitalized sectors.").

Remove it.

Signed-off-by: Dr. David Alan Gilbert <linux@treblig.org>
Link: https://lore.kernel.org/r/20240528215640.91771-1-linux@treblig.org
Reviewed-by: Himanshu Madhani <himanshu.madhani@oracle.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Daniel Wagner <dwagner@suse.de>
---
 drivers/scsi/qla2xxx/qla_isr.c |    6 ------
 1 file changed, 6 deletions(-)

--- a/drivers/scsi/qla2xxx/qla_isr.c
+++ b/drivers/scsi/qla2xxx/qla_isr.c
@@ -2885,12 +2885,6 @@ qla2x00_handle_sense(srb_t *sp, uint8_t
 	}
 }
 
-struct scsi_dif_tuple {
-	__be16 guard;       /* Checksum */
-	__be16 app_tag;         /* APPL identifier */
-	__be32 ref_tag;         /* Target LBA or indirect LBA */
-};
-
 /*
  * Checks the guard or meta-data for the type of error
  * detected by the HBA. In case of errors, we set the
